### ISSUE TITLE
Trying to use AVC444, works for video but not for anything else?

### ISSUE THREAD

**Issue by darkguy2008** [FreeRDP/FreeRDP#5430]
When using the latest nightly versions of wfreerdp.exe (for Windows) using AVC444 I get this behavior:

![ezgif com-optimize](https://user-images.githubusercontent.com/6304136/59344605-ad953f80-8ccb-11e9-89b7-a42ed39a49a5.gif)

Videos look OK I guess, I tried youtube and it was mostly fast, but everything else is just crap. What gives? Is there anything I can do? The AVC444 mode gives me the best speed I need to connect to the remote server to do frontend development work, so a fast UI is important. 

I have control over the machine shown in the GIF. I need to connect to another one inside that network, but I don't have control over that second one. Is there anything I can do in the machine I'm showing? Windows is latest 1903 build.

This is my commandline:

```
wfreerdp.exe +window-drag +fonts /bpp:8 /gfx:AVC444 /multitransport /network:auto
```

Thanks in advance!

**Comment by akallabeth** [comment]
nightly builds currently fall back to the build in windows media decoder for h264 but we seem to experience some issues with that one (depends a lot on the os used, so no pattern currently)
Providing a OpenH264 dll (just copy to where the .exe is) could help.
but from the looks of it there is some issue with the windows client itself.

Patches welcome, but windows client is currently kind of unmaintained :/

**Comment by darkguy2008** [comment]
@akallabeth thanks for your answer! Oh I see. I tried to put the OpenH264.dll library (tried with both x86 and x64) and it didn't do anything. It's too bad that the Windows version is unmaintained...

I got a hold of a macOS and installed xfreerdp through brew using --HEAD, but it's missing the h264 options. How do I get them to show? or is it the same story with OSX? I really need AVC444 working. Do you know a sure way how to? Ubuntu would?

**Comment by akallabeth** [comment]
Xfreerdp 2 or our nightly builds for linux should work. /buildconfig will tell if openh264 or ffmpeg is compiled in

**Comment by darkguy2008** [comment]
@akallabeth thanks, that worked. I ended up using Parallels and Ubuntu to use the latest xfreerdp2 binaries, it works, yeah, but eh... Parallels ain't free.

What I can confirm is that it works under Linux, could we get this working in OSX at least? Windows would be far better of course. If it works under Linux, what's holding it from working correctly in OSX or Windows?

**Comment by akallabeth** [comment]
We don't ship binaries (except nightly builds), but you can compile for these platforms.

**Comment by darkguy2008** [comment]
Well that's unfortunate. I could try to do a build but honestly, why make it so hard? I tried putting that DLL into the folder and it didn't do anything. Is there any non-compile way?

The nightly builds are broken too, both in Jenkins and the other 3rd-party, which is why I made the issue here. If you know a sure-fire way with clear instructions to compile on Windows and guarantee it'll work, I'm in.
